Packs/Wiz/IncidentFields/incidentfield-Wiz_Issue_Resolution_Recommendation.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "incident_wizissueresolutionrecommendation",
+    "version": -1,
+    "modified": "2022-08-24T14:22:56.074323111Z",
+    "name": "Wiz Issue Resolution Recommendation",
+    "ownerOnly": false,
+    "cliName": "wizissueresolutionrecommendation",
+    "type": "markdown",
+    "closeForm": true,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedToAll": true,
```

Comment:
Please associate the incident field to the relevant incident type.

Reply:
Done in new commit